openwisp_monitoring/device/api/views.py

```diff
@@ -200,6 +201,20 @@ def _write(self, request, pk):
                 metric.write(client['mac'])
             if created:
                 self._create_clients_graph(metric)
+        if 'resources' in data:
+            if 'load' in data['resources']:
+                self.resources_metric_write(
+                    pk, ct, key='load', value=100 * data['resources']['load'][1]
+                )
+            if 'disk_usage' in data['resources']:
+                usage = 0
+                for disk in data['resources']['disk_usage']:
```

Comment:
@nemesisdesign What do you think about this? Should we save the value aggregated or one value for each filesystem?

In any case, saving the sum of `disk['percent_used']` doesn't make much sense, at least `disk['percent_used'] / num_elements`